docs/howto/add_relationship_links.md

```diff
@@ -104,12 +104,7 @@ class Api::V1::UserSerializer < ActiveModel::Serializer
 
   has_many :microposts, serializer: Api::V1::MicropostSerializer do
     link(:related) { api_v1_microposts_path(user_id: object.id) }
-  end
-
-  #this is needed to avoid n+1, gem core devs are working to remove this necessity
-  #more on: https://github.com/rails-api/active_model_serializers/issues/1325
-  def microposts
-    object.microposts.loaded ? object.microposts : object.microposts.none
+    include_data false
```

Comment:
For this specific comment, I think it is no longer needed. `include_data false` is already documented elsewhere as avoiding the N+1/loading the data.

That said, we should probably rework this section. It's totally valid to have both `data` and `links`, so there's really no inherent reason to have `include_data false` in the documentation example in the first place. We're just talking about link generation here.

Instead, I'd add a section below this (or elsewhere) on conditionally including `data`, how and why one would do this. @ksamc any interest in adding this documentation, based on the explanation in https://github.com/rails-api/active_model_serializers/pull/1931? Otherwise I will submit a separate PR.